core/src/main/java/tc/oc/pgm/teams/Team.java

```diff
@@ -37,8 +37,6 @@
   private TeamMatchModule tmm;
   private JoinMatchModule jmm;
   protected @Nullable String name = null;
-  protected @Nullable Component componentName;
```

Comment:
We should be caching the `Componet` name. Instead, listen to the team rename event and change it.